src/app/utils/BountyUtils.ts

```diff
@@ -313,23 +325,30 @@ const BountyUtils = {
             fields.push({ name: 'For user', value: assignedUser.user.tag, inline: false })
         }
 
-        let footer = {};
+        let footerArr = [];
+        if (bounty.tags?.channelCategory) {
+            footerArr = footerArr.concat(bounty.tags.channelCategory);
+        }
+        if (bounty.tags?.text) {
+            footerArr = footerArr.concat(bounty.tags.text)
+        }
+        let footer = { text: footerArr.length ? `🔖${footerArr.join(' 🔖')}\n \n` : ''};
```

Comment:
Can we put a limit on the size here? If we have long or many tags, might hit a character limit